test/rebuild.test.js

```diff
@@ -21,47 +24,42 @@ test('rebuild (I am added to a group)', t => {
     }
   }
 
-  replicate({ from: admin, to: me, name, live: true })
-
-  /* set up listener */
-  me.rebuild.hook(function (rebuild, [cb]) {
-    t.pass('I automatically call a rebuild')
-
-    rebuild(() => {
-      cb && cb()
-      t.true(me.status().sync.sync, 'all indexes updated') // 2
-
-      pull(
-        me.createUserStream({ id: admin.id, private: true }),
-        pull.drain(
-          m => {
-            t.equal(typeof m.value.content, 'object', `I auto-unbox msg: ${m.value.content.type}`)
-          },
-          (err) => {
-            if (err) throw err
-            admin.close()
-            me.close()
-            t.end()
-          }
-        )
-      )
-    })
-    t.false(me.status().sync.sync, 'all indexes updating') // 1
-  })
-
   /* kick off the process */
-  admin.tribes.create({}, (err, data) => {
-    if (err) throw err
+  const data = await p(admin.tribes.create)({})
+
+  await p(admin.tribes.invite)(data.groupId, [me.id], { text: 'ahoy' })
+
+  await replicate({ from: admin, to: me, name })
+
+  // time for rebuilding
+  await p(setTimeout)(500)
+
+  t.true(me.status().sync.sync, 'all indexes updated')
```

Comment:
new pattern I've been using for this sorta wait : 
```suggestion
  while (!me.status().sync.sync) await p(setTimeout)(100)
  t.pass('all indexes updated')
```